### PR TITLE
Events Page

### PR DESCRIPTION
**Pull request by Chaitanya134**
1) Created the basic UI as mentioned in the wireframes
2) Functionalities to select and update events based on categories not added
3) Dependencies Added and Used - React Icons (for icons), React Swiper (for carousel)

### CODE REVIEW

File: src/Pages/Events/index.jsx [hunk 1]
```diff
@@ -1,69 +1,125 @@
-import React from "react";
-import Card from "./card";
-import styles from "./events.module.css";
+import React, { useRef } from "react";
+import Card from "./Card";
```
Comment (adnanhashmi09) [inline src/Pages/Events/index.jsx:2]: The import statement here is wrong. Change from `./Card` to `./card`.

**Comment by adnanhashmi09** [comment]
- Text is too small in the modal. Please refer to the design and make the necessary changes.
-  Make the height of the modal adaptive to its content. Extra whitespace should be avoided.

**Comment by Chaitanya134** [comment]
Did the necessary changes. Kindly review

File: src/Pages/Events/index.jsx (hunk 1 above)
Comment (adnanhashmi09) [reply src/Pages/Events/index.jsx:2]: @Chaitanya134 This issue hasn't been resolved yet.

**Comment by adnanhashmi09** [comment]
@Chaitanya134 dude the import problem still persists. I have even added a comment in the review. Seems like you missed it.

**Comment by adnanhashmi09** [comment]
1) The spacing of the content of the modal is off. We need more padding and gap. Please refer to the design.
2) Give the modal a `max-width` of something around 1024px.

**Pull request merged**